docs/scripts.rst

```diff
@@ -9,37 +9,111 @@ Benchpark Scripts
+         {
+               "spec": "cmake@3.23.1",
+               "prefix": "/usr/tce/packages/cmake/cmake-3.23.1",
+         }
+      ],
+      "buildable": False,
+   },
 
+The script will appropriately identify the change to the package version:
+
+.. code-block:: console
+
+   $ benchpark-python diffSystems.py -n changeCMakeBranch -s llnl-sierra
+
+   llnl-sierra
+        llnl-sierra/system_id.yaml
+                The YAML files benchpark-old/llnl-sierra/system_id.yaml and benchpark-new/llnl-sierra/system_id.yaml are identical.
+        llnl-sierra/software.yaml
+                The YAML files benchpark-old/llnl-sierra/software.yaml and benchpark-new/llnl-sierra/software.yaml are identical.
+        llnl-sierra/variables.yaml
+                The YAML files benchpark-old/llnl-sierra/variables.yaml and benchpark-new/llnl-sierra/variables.yaml are identical.
+        llnl-sierra/auxiliary_software_files/packages.yaml
+                The YAML files benchpark-old/llnl-sierra/auxiliary_software_files/packages.yaml and benchpark-new/llnl-sierra/auxiliary_software_files/packages.yaml are different. Here are the differences:
+                  {'values_changed': {"root['packages']['cmake']['externals'][0]['prefix']": {'new_value': '/usr/tce/packages/cmake/cmake-3.23.1',
+                                                                                             'old_value': '/usr/tce/packages/cmake/cmake-3.29.2'},
+                                    "root['packages']['cmake']['externals'][0]['spec']": {'new_value': 'cmake@3.23.1',
+                                                                                          'old_value': 'cmake@3.29.2'}}}
+        llnl-sierra/auxiliary_software_files/compilers.yaml
+                The YAML files benchpark-old/llnl-sierra/auxiliary_software_files/compilers.yaml and benchpark-new/llnl-sierra/auxiliary_software_files/compilers.yaml are identical.
+
+.. note::
+   This script *does not* need to be ran on the target system to work correctly, e.g. the above example for ``llnl-sierra`` can be ran from your local machine.
 
 Compare Experiment Builds
 -------------------------
 ``lib/scripts/diffExperiments.py``
 
-Compare experiment builts between two versions of benchpark (leverages ``altdiff.py``).
-This is helpful to compare changes to either ``experiment.py`` or ``system.py``, as changes to either may result in a different experiment build.
-This requires the user to specify which ``system`` and ``cluster`` they are running on. For example, running on Ruby::
+This script enables the user to compare the results of initializing an ``experiment.py``
+from ``experiment init ...`` from two commits in benchpark.  experiment builts between two versions of benchpark (leverages ``diffSpecs.py``).
 
-    benchpark-python diffExperiments.py -s llnl-cluster -c ruby -p openmp
+Question to Michael: why are you mentioning system.py here?  system.py and experiment.py get mapped by ``benchpark setup`` and not before.
```

Comment:
This script builds the benchmark (``ramble workspace setup`` so beyond `benchpark setup`) and checks the spack builds with ``diffSpecs.py``. It was named ``diffExperimentBuilds.py`` at one point, maybe that is the less confusing name. 

`diffExperiments.py` should be a separate script that just checks the output of `experiment init` then. I can make this pretty easily.